File: Front/FFStats-App/src/app/Model/Estatistica.ts
import { Jogador } from "./Jogador";
import { Partida } from "./Partida";

export interface Estatistica {
  id: number;
  partidaId: number;
  partida: Partida;
  jogadorId: number;
  jogador: Jogador;
  kill: number;
  morte: number;
  assistencia: number;
  dano: number;
  derrubado: number;
  cura: number;
  levantados: number;
  ressucitou: number;
  tempo: string;
}

File: Back/src/FFStats.Domain/Models/Estatistica.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace FFStats.Domain.Models
{
    [Table("tb_estatistica")]
    public class Estatisticas
    {
        public int id { get; set; }
        public int partidaId { get; set; }
        public Partida Partida { get; set; }
        public int jogadorId { get; set; }
        public Jogador Jogador { get; set; }
        public int Kill { get; set; }
        public int Morte { get; set; }
        public int Assistencia { get; set; }
        public int Dano { get; set; } 
        public int Derrubado { get; set; } 
        public int Cura { get; set; } 
        public int Levantados { get; set; } 
        public int Ressucitou { get; set; }
        public int Tempo { get; set; }
    }
}